src/schema/mutations/products.mutations.ts

```diff
@@ -0,0 +1,18 @@
+import { GraphQLFieldConfig, GraphQLNonNull } from "graphql"
+import { ProductService } from "../../services/products.service"
+import productInput from "../types/product.input"
+import { ProductType } from "../types/product.type"
+
+const createProduct: GraphQLFieldConfig<any, any> = {
+  type: ProductType,
+  args: { input: { type: new GraphQLNonNull(productInput) } },
+  resolve: async function (_source, { input }) {
+    const product = await ProductService.createProduct(input)
+
+    return product
```

Comment:
just return the promise, probably in the future you will need to use `class-transformer`

Reply:
Done